From f40143f7acca81b7d39d774ed4c349aec8d9310b Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Sat, 23 Sep 2017 11:37:23 +0200
Subject: [PATCH] Form.cc: include ctype.h to fix build error

Fixes
Form.cc:546:28: error: 'isdigit' was not declared in this scope
Form.cc:548:34: error: 'isxdigit' was not declared in this scope
Form.cc:575:40: error: 'isxdigit' was not declared in this scope

detected by buildroot autobuilders:
http://autobuild.buildroot.net/results/a6a/a6a336f8b6a0136b204a34091e33dc4598178125/
when cross-compiling with
gcc version 4.7.3 (crosstool-NG hg+-c65fcf8a34b7)

Patch sent upstream: https://bugs.freedesktop.org/show_bug.cgi?id=102951

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 poppler/Form.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/poppler/Form.cc b/poppler/Form.cc
index 83bceb20..e7efae95 100644
--- a/poppler/Form.cc
+++ b/poppler/Form.cc
@@ -32,6 +32,7 @@
 #include <stddef.h>
 #include <stdlib.h>
 #include <string.h>
+#include <ctype.h>
 #include "goo/gmem.h"
 #include "goo/GooString.h"
 #include "Error.h"
-- 
2.11.0

